src/tests/multihost/ipa/test_adtrust.py

```diff
@@ -731,3 +731,62 @@ def test_skip_members_in_view_search(multihost):
         assert f"group1@{domain}" in lookup_id_client.stdout_text, f"group1 name was not resolved" \
                                                                    f" on ipa-client."
         assert f"admins" in lookup_id_client.stdout_text, "admins name was not resolved on ipa-client."
+
+    @staticmethod
+    def test_s2n_get_request(multihost):
+        """
+        :title: User lookup on IPA client fails with 's2n get_fqlist request failed'
+        :id: fae73dd7-5a18-4aa4-a39f-a20a2f66b1c9
+        :customerscenario: true
+        :bugzilla:
+            https://bugzilla.redhat.com/show_bug.cgi?id=2192708
+            https://bugzilla.redhat.com/show_bug.cgi?id=2196838
+            https://bugzilla.redhat.com/show_bug.cgi?id=2196816
+            https://bugzilla.redhat.com/show_bug.cgi?id=2196839
+        :description: When checking if the input group-name is the original name from AD or an
+            overwritten one the comparison is currently done case sensitive. Since AD handles names
```

Comment:
Hi,

maybe it would be worth to be a bit more specific here. It cannot be any group but has to be the user-private-group of an AD user with a mixed-case name.

Otherwise the test steps are looking good.

bye,
Sumit

Reply:
updated.

Reply:
updated.